Homework1 -- MADE

NikOrlov opened the pull request:
По ошибке первую дз загрузил не в ту организацию (ссылка на предыдущий PR: https://github.com/made-ml-in-prod-2022/NikOrlov/pull/1)

Решение задачи классификации, реализован пайплайн обучения и предсказания. Обучение настраивается через конфиги (за основу был взят репозиторий Михаила)
Самооценка - 1 балл
EDA - 1 балл
Функция для тренировки - 3 балла
Функция для предсказания - 3 балла
Модульная структура - 2 балла
Логгеры - 2 балла
Тесты - 3 балла
Синтетические данные - 2 балла
Конфиги в yaml формате - 3 балла
В кофигах датаклассы - 2 балла
Зависимости в requirements.txt - 1 балл
Настройка CI - 3 балла
Итого: 26 баллов

NikOrlov committed:
added all data

NikOrlov committed:
renamed README

SVizor42 commented on ml_project/src/fit_predict.py:
```diff
@@ -0,0 +1,61 @@
+from sklearn.metrics import roc_auc_score, accuracy_score, f1_score
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import Pipeline
+from src.entities import TrainingParams
```
Разделить пустой строкой стандартные и локальные импорты

NikOrlov replied:
Сделано

SVizor42 commented on ml_project/src/fit_predict.py:
```diff
@@ -0,0 +1,61 @@
+from src.entities import TrainingParams
+
+
+SklearnModel = Union[LogisticRegression]
```
Forest?

NikOrlov replied:
Не заметил, добавил форест

SVizor42 commented on ml_project/tests/data_generator.py:
```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+from tests.fake_data import FakeDatasetBuilder
```
Локальные импорты следуют после стандартных пакетов

NikOrlov replied:
затупил, забыл pylint-ом прогнать, поправил

SVizor42 reviewed (approved):
Сама работа была сдана вовремя, просто попала в старую оргу. 
Нужно только добавить чекпойнт файлы у ноутбука в `gitignore`.

**Сумма: 27 баллов** (пункт 0 также оценивается) 

Все хорошо, ветку можно сливать

SVizor42 commented:
Не забудь добавить метки `MADE` и `hw1`

NikOrlov committed:
fixed fit_predict.py

added blank line between imports, added Forest too typing standard and local imports

NikOrlov committed:
Update data_generator.py
